frontend/src/components/utils/Link.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+import React from "react";
+import { Mixpanel } from "../../../mixpanel/index";
+
+interface Props {
+  href: string;
+  as?: string;
+  children: React.ReactNode;
+}
+
+export default ({ href, as, children }: Props) => {
+  const clicked = () =>
+    Mixpanel.track("Click Link", { href: href, as: as ? as : "" });
+  return (
+    <div onClick={clicked}>
```

Comment:
`div` is a block-displaying element, which breaks text when a link is inline, e.g. the text message on the [transaction row](https://near-explorer-frontend-pr-556.onrender.com/transactions):

![image](https://user-images.githubusercontent.com/304265/108757754-0735b000-7529-11eb-8978-fd1c3fa21adc.png)

Consider using `span` instead